Description: nall/Makefile fixes

* Use the flags from CFLAGS, CPPFLAGS and LDFLAGS. They are set in the rules file.

Author: Tobias Hansen <thansen@debian.org>

--- a/nall/Makefile
+++ b/nall/Makefile
@@ -44,18 +44,15 @@
     compiler := clang++
     flags := -w -stdlib=libc++
     link := -lc++ -lobjc
-  else ifeq ($(platform),bsd)
-    compiler := clang++
-    flags := -w -I/usr/local/include
   else
     compiler := g++
     flags :=
-    link :=
+    link := $(LDFLAGS)
   endif
 
-  cflags := -x c -std=c99
+  cflags := -x c -std=c99 $(CFLAGS) $(CPPFLAGS)
   objcflags := -x objective-c -std=c99
-  cppflags := -x c++ -std=c++11
+  cppflags := -x c++ -std=c++11 $(CFLAGS) $(CPPFLAGS)
   objcppflags := -x objective-c++ -std=c++11
 endif
 
